### DISCUSSION TITLE
(xApps) RIC Subscription Request not sending messages.

### DISCUSSION THREAD

**Discussion by yucongli1234** [srsran/srsRAN_Project#793] [General Help]
Hey, currently my setup is SRSRAN with B210 as the radio, and I got everything to work, but the xApp.. I'm also currently using the FlexRIC with a real UE. The UE attaches fine, but when I run the xApp, I'm not recieving any RIC Subscription Request. 
![open5gc](https://github.com/user-attachments/assets/68195d75-32a1-4f45-b24e-67721791796c)
![image](https://github.com/user-attachments/assets/4afc8e53-02dd-48df-ba04-df4f5d66a84a)
![image](https://github.com/user-attachments/assets/71a19251-a4fa-4f8f-8e3c-a7b44244a608)
![image](https://github.com/user-attachments/assets/2a574068-4dff-4461-ab39-db964412ad79)

**Comment by pgawlowicz** [comment]
could you reduce log level and share your logs?
